TestModels/Resource/runtimes/rust/src/conversions/get_resource_data/_get_resource_data_input.rs

```diff
@@ -0,0 +1,39 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+#[allow(dead_code)]
+pub fn to_dafny(
+    value: crate::operation::get_resource_data::GetResourceDataInput,
+) -> ::std::rc::Rc<
+    ::simple_resources_dafny::r#_simple_dresources_dinternaldafny_dtypes::GetResourceDataInput,
+> {
+    ::std::rc::Rc::new(::simple_resources_dafny::r#_simple_dresources_dinternaldafny_dtypes::GetResourceDataInput::GetResourceDataInput {
+        stringValue: dafny_standard_library::conversion::ostring_to_dafny(value.string_value()),
+        booleanValue: dafny_standard_library::conversion::obool_to_dafny(value.boolean_value()),
+        integerValue: dafny_standard_library::conversion::oint_to_dafny(value.integer_value()),
+        longValue: dafny_standard_library::conversion::olong_to_dafny(value.long_value()),
+        blobValue: dafny_standard_library::conversion::oblob_to_dafny(value.blob_value())
```

Comment:
Love the conversion helper methods btw :)

Reply:
Thanks.